### ISSUE TITLE
.NET Core July Update - 2.1.2

### ISSUE THREAD

**Issue by leecow** [dotnet/core#1765]
#.NET Core July Update - 2.1.2

[Release Notes](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1.2.md)
[Download 2.1.2](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.1.2-download.md)

Included fixes address [CVE-2018-8356: .NET Core Security Feature Bypass Vulnerability](https://github.com/dotnet/announcements/issues/73)

Please report any issues you find with 2.1.2, either responding to this issue, creating a new issue or creating a new issue in one of the following repos:

* [aspnet/Home](https://github.com/aspnet/Home) - for ASP.NET Core questions
* [dotnet/cli](https://github.com/dotnet/cli/issues) - for CLI tools and questions
* [dotnet/corefx](https://github.com/dotnet/corefx/issues) - for API issues and questions
* [dotnet/coreclr](https://github.com/dotnet/coreclr/issues) - for runtime issues
* [dotnet/core-setup](https://github.com/dotnet/core-setup/issues) - for installer issues
* [nuget/home](https://github.com/nuget/home/issues) - for NuGet questions and issues

**Comment by Scandal-UK** [comment]
On CentOS running ```yum update``` fails with the following error.

```
Error: Package: dotnet-runtime-2.1-2.1.2-1.x86_64 (packages-microsoft-com-prod)
           Requires: dotnet-runtime-deps-2.1 >= 2.1.2
           Installed: dotnet-runtime-deps-2.1-2.1.1-1.x86_64 (@packages-microsoft-com-prod)
               dotnet-runtime-deps-2.1 = 2.1.1-1
           Available: dotnet-runtime-deps-2.1-2.1.0-1.x86_64 (packages-microsoft-com-prod)
               dotnet-runtime-deps-2.1 = 2.1.0-1
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by dncAdmin** [comment]
vs2017 17.5 new project shows wrong version of .net core 2.1.2

After installing .net core SDK 2.1.302 with Runtime 2.1.2 ,

vs2017 17.5 -> new project -> asp.net core project still shows the version of 2.1.1 and 2.1.0 in the Nuget and SDK dependence.

it should be 2.1.2

![Uploading image.png…]()

PC:

Win10

**Comment by leecow** [comment]
@mrbigtities and @Scandal-UK - let me know if you're still having issues with CentOS installs.

**Comment by beylkhanovdamir** [comment]
@leecow why VS v.15.7.5 does not comprise the .net core sdk 2.1.2 already on board? I went to the [ms website](https://www.microsoft.com/net/download/dotnet-core/2.1) to download the latest .net core sdk and only then my problem was gone

**Comment by leecow** [comment]
@beylkhanovdamir - The update cycles into VS are a bit different. The 2.1 runtime and supporting SDK will be in VS 15.8 (and already in the Previews which VS has been releasing).

**Comment by leecow** [comment]
2.1.3 released. Closing this issue.

cc @vivmishra
